README.md

```diff
@@ -2,14 +2,26 @@ _NOTE: The interfaces of flux-accounting are being actively developed and are no
 
 ## flux-accounting
 
-Development for a bank/accounting interface for the Flux resource manager. Writes and saves job step history, user account information, and priority calculation to persistent storage using Python's SQLite3 API. 
+Development for a bank/accounting interface for the Flux resource manager. Writes and saves user account information to persistent storage using Python's SQLite3 API.
 
-##### job step history
+##### Install instructions
 
-Running as a cron job, `write_jobs.py` uses a flux-core RPC to retrieve job data that have recently transitioned to an INACTIVE state in the past _x_ minutes. It parses the job record data and writes it to a SQLite database, which can be queried at a later time. 
+This repository uses `pandas` for data display information. To install this dependency, along with any other packages specified in **requirements.txt**, and also run unit tests, you can run `make check`:
```

Comment:
Thanks @cmoussa1. Strictly speaking, run tests and make check are not part of Install instructions? 

Would it make sense to have "build instructions", "test instructions" and "install instructions"? And discuss the step for each in different section? 

Is `pandas` a test program dependency or a dependency for the core flux-accounting?

Reply:
Thanks for the feedback @dongahn. I believe what's required as far as install instructions after pulling down this repository are the following (let me know if you had to do anything else on your end):

- have `python3` installed in your environment (I used `brew install python3`)
- have `pip3` installed in your environment (which should be included when installing `python3`)
- install the dependencies listed in **requirements.txt** (which for now is just `pandas`) with `pip3 install -r requirements.txt`

`pandas` is a dependency for the core flux-accounting.